src/Fixer/ClassNotation/SingleClassElementPerStatementFixer.php

```diff
@@ -144,7 +142,9 @@ private function expandElement(Tokens $tokens, $startIndex, $endIndex)
             if ($divisionContent) {
                 $tokens->insertAt($i + 1, new Token(array(T_WHITESPACE, $divisionContent)));
             }
-            $tokens->insertAt($i + 2, $modifiers);
+            // collect modifiers
+            $sequence = $this->getModifiersSequences($tokens, $startIndex, $endIndex);
```

Comment:
I'm not sure that calling `getModifiersSequences` method in loop is good, as it's already iterate through tokens in loop as well